Makefile

```diff
@@ -1,41 +1,80 @@
 SHELL := /bin/bash
+include functions.mk
+
+# Name of the exporter
+EXPORTER_NAME := ebs-iops-reporter
```

Comment:
Really can't be hard coded if expect to generate for many exporters, have to loop over them?